book/src/drivers.md

```diff
@@ -60,6 +60,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Intel PMEM-CSI](https://github.com/intel/pmem-csi) | `pmem-csi.intel.com` | v1.0 | A Container Storage Interface (CSI) driver for [PMEM](https://pmem.io/) from Intel | Persistent and Ephemeral | Read/Write Single Pod | Yes | Raw Block
 [JuiceFS](https://github.com/juicedata/juicefs-csi-driver) | `csi.juicefs.com` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for JuiceFS File System | Persistent | Read/Write Multiple Pod | Yes | 
 [kaDalu](https://github.com/kadalu/kadalu) | `org.kadalu.gluster` | v0.3 | A CSI Driver (and operator) for GlusterFS | Persistent | Read/Write Multiple Pods | Yes | 
+[KumoScale Block Storage](https://github.com/KioxiaAmerica/kumoscale-csi) | `kumoscale.kioxia.com` | v1.0 | A Container Storage Interface (CSI) Driver for KumoScale Block Storage | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Snapshot, Expansion, Thin Provisioning, Volume Migration
```

Comment:
"Read/Write Multiple Pods" implies that a single "KumoScale Block Storage" can be accessed (read/write) from multiple pods (potentially from different nodes) at the same. I want to confirm that is the case (usually block devices don't allow this as it requires multi-attach and a shared filesystem).

Reply:
you are correct, I will make the correction